pyproject.toml

```diff
@@ -30,14 +30,15 @@ classifiers = [
   "Typing :: Typed",
 ]
 dynamic = ["version"]
-dependencies = ["pybamm"]
+dependencies = ["pybamm", "cookiecutter"]
 
 [project.optional-dependencies]
```

Comment:
I would suggest changing `nox` in this table to `nox[uv]`, otherwise it will always use `virtualenv`.